studio/tests/pages/projects/reports/api-report.test.js

```diff
@@ -3,8 +3,12 @@ import { render } from '../../../helpers'
 import { waitFor, screen } from '@testing-library/react'
 import { ApiReport } from 'pages/project/[ref]/reports/api-overview'
 import userEvent from '@testing-library/user-event'
+import * as useFillTimeseriesSortedModule from 'hooks/analytics/useFillTimeseriesSorted'
 
 beforeEach(() => {
+  // This hook can result in a huge amount of data being loaded into memory (array of 130k+ objects)
+  jest.spyOn(useFillTimeseriesSortedModule, 'default').mockReturnValue([])
```

Comment:
the reason for this is because we try to auto-detect the timestamp truncation, and as the tests are mocked with timestamps that provide second-ly timestamps, it expands the mocked data to fill in second-based timestamps as well.
it would be safer to mock this hook globally, as there are many areas in the app that would use this function.

Reply:
@brandongregoryscott could you move this mock to the `__mocks__/hooks.js`?

Reply:
Can do! Makes sense to globally mock so it doesn't accidentally crash more test suites as they are added.